portfolio/pom.xml

```diff
@@ -23,6 +23,11 @@
       <version>4.0.1</version>
       <scope>provided</scope>
     </dependency>
+	<dependency>
+  	  <groupId>com.google.code.gson</groupId>
+  	  <artifactId>gson</artifactId>
+  	  <version>2.8.6</version>
+	</dependency>
```

Comment:
nit: Let's un-indent this a bit so it lines up with the other `<dependency>`s.

Reply:
fixed

Reply:
nit: Seems like the dependencies still have a little bit of funky indentation. Ideally we have something like:

```
<dependencies>
  <dependency>
    <groupId>javax.servlet</groupId>
    <artifactId>javax.servlet-api</artifactId>
    <version>4.0.1</version>
    <scope>provided</scope>
  </dependency>
  <dependency>
    <groupId>com.google.code.gson</groupId>
    <artifactId>gson</artifactId>
    <version>2.8.6</version>
  </dependency>
  <dependency>
    <groupId>com.google.appengine</groupId>
    <artifactId>appengine-api-1.0-sdk</artifactId>
    <version>1.9.59</version>
  </dependency>
</dependencies>
```

This isn't critical or anything. It's just nice to have well-indented code :-)

Reply:
Fixed, thanks. This is slightly odd, it doesn't look like the fixes I made were entirely reflected here.